locales/en-US/editor.properties

```diff
@@ -4,8 +4,12 @@
 
 # General File I/O Error strings
 
+# Dialog Headers
+ERROR_DIALOG_HEADER=Error
```

Comment:
Is this really necessary?  Isn't there already a string we could use, or doesn't Brackets have some way to title an error dialog?